doc/man7/provider-cipher.pod

```diff
@@ -162,7 +162,7 @@ context in the I<cctx> parameter that should have been previously initialised vi
 a call to OP_cipher_encrypt_init() or OP_cipher_decrypt_init.
 This should call the raw underlying cipher function without any padding.
 This will be invoked in the provider as a result of the application calling
-L<EVP_Cipher(3)>.
+EVP_Cipher().
```

Comment:
The link is correct, the documentation is lacking.  So no.

(althought if it were me, I would remove `EVP_Cipher()` entirely...  but we can't, because backward compat)

Reply:
It could be deprecated, but it's used internally.  Fix that. :)
Will not continue the "keep the link to unwritten manpages" discussion.  Perhaps I should open an issue to put it in one place.

Reply:
Since this discussion, `EVP_Cipher` has appeared in `EVP_EncryptInit.pod`, so by now you should be able to restore this link without feeling that you're violation any sense of "there shall be no dangling links"